go.mod

```diff
@@ -1,9 +1,9 @@
-module github.com/agilebits/redicrypt
+module github.com/cohix/redicrypt
 
 go 1.14
 
 require (
-	github.com/gomodule/redigo v2.0.0+incompatible
```

Comment:
Sorry are we downgrading? 🤔

Reply:
I think we're actually upgrading:
<img width="567" alt="Screen Shot 2021-03-25 at 2 08 15 PM" src="https://user-images.githubusercontent.com/11049609/112544013-c1604780-8d73-11eb-8e99-884fc4a67ef7.png">

Reply:
I wasn't sure, so I checked and found this answer on [stackoverflow](https://stackoverflow.com/a/57356777):

> +incompatible means the dependency has a semver major version of 2 or higher and is not a Go module yet (it doesn't have go.mod in its source code).

Reply:
Ah I understand. Thank you for clarifying!